src/main/java/com/minecolonies/MineColonies.java

```diff
@@ -18,12 +19,16 @@
 import net.minecraftforge.fml.common.network.NetworkRegistry;
 import net.minecraftforge.fml.common.network.simpleimpl.SimpleNetworkWrapper;
 import net.minecraftforge.fml.relauncher.Side;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.jetbrains.annotations.NotNull;
 
 @Mod(modid = Constants.MOD_ID, name = Constants.MOD_NAME, version = Constants.VERSION,
   dependencies = Constants.FORGE_VERSION, acceptedMinecraftVersions = Constants.MC_VERSION)
 public class MineColonies
 {
+    private static Logger logger = LogManager.getLogger(Constants.MOD_ID);
```

Comment:
This isn't going to fix our issue, the unit tests fail when LogManager is loaded.